list2/ex36/index.js

```diff
@@ -0,0 +1,38 @@
+import promptSync from "prompt-sync";
+const prompt = promptSync();
+
+const MAX_NUMBERS = 13;
+const BETTING_PLAYERS = 100;
+
+function luckyLotteryNumbers() {
+  const luckyNumbers = [];
+  for (let i = 0; i < MAX_NUMBERS; i++) {
+    luckyNumbers.push(Math.floor(Math.random() * 100) + 1);
+  }
+  return luckyNumbers;
+}
```

Comment:
The luckyLotteryNumbers function generates random numbers between 1 and 100. However, duplicates might occur, which is not typical for a lottery.